python_src/pyproject.toml

```diff
@@ -27,10 +27,15 @@ psutil = "^5.9.1"
 schedule = "^1.1.0"
 alembic = "^1.10.2"
 types-pytz = "^2023.3.0.1"
+
+[tool.poetry.group.tableau]
```

Comment:
add dependency group for tableau dependencies. its not optional, meaning a user will have to change behavior to build without it.